modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb

```diff
@@ -23,63 +23,67 @@
 </div>
 
 <%= labelled_tabular_form_for @oauth_application, url: '/' do |f| -%>
-  <div class="form--field">
-    <label class="form--label"><%= t('storages.label_oauth_client_id') %></label>
+  <section class="form--section">
+    <fieldset class="form--fieldset">
```

Comment:
It does not make any sense to have a `form--fieldset`  and a `form--section` just below each other. I honestly don't know the difference between these two because they share a lot of styles and look quite similar 🤷‍♀️ . So just decide for one.